scripts/dnd5e-helpers.js

```diff
@@ -791,13 +790,14 @@ function ReactionRemove(currentToken) {
 
 }
 
-function OpenWounds() {
+function OpenWounds(actor, woundType) {
```

Comment:
since the only use of 'actor' is to get 'data.name'. lets just pass in the actor's name directly.
```suggestion
function OpenWounds(actorName, woundType) {
```